terraform/main.tf

```diff
@@ -0,0 +1,139 @@
+terraform {
+  backend "gcs" {
+    bucket = "f3441e415e6e5e7d-bucket-tfstate"
+    prefix = "terraform/state"
+  }
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.51.0"
+    }
+  }
+}
+
+variable "project_id" {
+  type    = string
+  default = "catalyst-cooperative-pudl"
+}
+
+variable "catalyst_people" {
+  type = list(string)
+  default = [
+    "trenton.bush@catalyst.coop",
+    "katherine.lamb@catalyst.coop",
+    "zach.schira@catalyst.coop",
+    "dazhong.xia@catalyst.coop"
+  ]
+}
+
+provider "google" {
+  project = var.project_id
+  region  = "us-east1"
+  zone    = "us-east1-c"
+}
+
+resource "google_service_account" "mozilla_dev_sa" {
+  account_id   = "mozilla-dev-sa"
+  display_name = "Mozilla dev"
+}
+
+resource "random_id" "bucket_prefix" {
+  byte_length = 8
+}
+
+resource "google_storage_bucket" "sec_10ks" {
+  name          = "${random_id.bucket_prefix.hex}-bucket-sec-10ks"
+  location      = "US"
+  storage_class = "STANDARD"
+  versioning {
+    enabled = true
+  }
+}
+
+resource "google_storage_bucket" "tfstate" {
+  name          = "${random_id.bucket_prefix.hex}-tfstate"
```

Comment:
So if we create a new Mozilla project, we'll need to actually follow [this guide](https://cloud.google.com/docs/terraform/resource-management/store-state#create_the_bucket) which basically says:

1. use a block such as this one to create a bucket to hold tfstate
2. **only then** do you configure the tfstate to be remote like in lines 1-4 - though point the gcs backend at the new bucket

If you want to stay in the PUDL project because the billing isn't separate, you can avoid making this new bucket and just use what we have already set up (the `f344...` bucket).